webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -470,6 +465,17 @@ def update_corr_param_resp(
         parent.save_state("active_correlation", corr_param_resp)
         return corr_param_resp
 
+    def get_selected_ensembles_from_store(
```

Comment:
The function should be "private" I think -> `_get_selected_ensembles_from_store` also it is a good idea to define it before it is used so at the beginning of this file maybe

Reply:
thank you, of courese, gonna make it private! 
about the beginning or end - this is i guess something we should discuss @hnformentin :sweat_smile:

Reply:
Regarding vertical ordering, we follow the recommendation in "Clean Code" 📖 : 
<h1>Vertical Ordering</h1>
In general we want function call dependencies to point in the downward direction. That is, a function that is called should be below a function that does the calling*.

As in newspaper articles, we expect the most important concepts to come first, and we expect them to be expressed with the least amount of polluting detail. We expect the low-level details to come last. This allows us to skim source files, getting the gist from the first few functions, without having to immerse ourselves in the details.

* This is the exact opposite of languages like Pascal, C and C++ that enforce functions to be defined, or at least declared, _before_ they are used.

Reply:
I was not aware of the "clean code" recommendation, can you also provide a link to where this is coming from? 

Anyway, then I think there are a lot of ppl in scout with a C/C++ background, myself included because most of the helper functions are defined at the top of files in most scout projects,  but I guess the bottom of the file is just as fine, not that it really matters if we are talking about a file that is 700+ lines.

Reply:
@DanSava it is the book called "Clean Code, a handbook of agile software craftsmanship", by Robert Martin, page 84. We have a printed sample at the office. I can ask in the channel to see what we want as a team.

Reply:
discussing in the team is a good idea, then we can find a convention we can follow : )

Reply:
made private!